### ISSUE TITLE
do/end for multi-line block misses some

### ISSUE THREAD

**Issue by mockdeep** [rubocop/rubocop#776]
Here's an example where the `Blocks` cop fails to catch a multi-line violation:

``` ruby
expect do
  campaign_invite.send!
end.to change {
  user.password_reset_expires_at
}.from(nil).to(instance_of(ActiveSupport::TimeWithZone))
```

It caught the first one, but the `change {}` block doesn't get caught by rubocop.
